purefa/README.md

```diff
@@ -0,0 +1,123 @@
+# Agent Check: PureFA
+
+## Overview
+
+This check monitors the [Pure Storage FlashArray][1] through the [Datadog Agent][2] and the [Pure Storage Prometheus exporter][1].
```

Comment:
Should "Pure Storage FlashArray" and "Pure Storage Prometheus exporter" point to different links?

Reply:
No - will fix